requirements/dev.in

```diff
@@ -1,7 +1,4 @@
 -e git+https://github.com/kytos-ng/python-openflow.git#egg=python-openflow
--e git+https://github.com/kytos-ng/kytos.git#egg=kytos
 -e git+https://github.com/kytos-ng/of_core.git#egg=kytos_of_core
--e git+https://github.com/kytos-ng/sdntrace.git#egg=amlight-sdntrace
--e .[dev]
-pytest-asyncio==0.18.3
-black==22.3.0
+-e git+https://github.com/kytos-ng/kytos.git@feat/starlette#egg=kytos[dev]
```

Comment:
Once `kytos-ng/kytos` `feat/starlette` branch lands, I'll revert this branch here.